spec/signalfx-transform-helper-spec.js

```diff
@@ -25,18 +24,17 @@ describe('signalfx-transform-helper', () => {
     expect(flattened['key_otherNestedKey']).to.be.equal('otherValue');
   });
 
-  it('should handle different levels of nesting', () => {
+  it('should handle exactly two levels of nesting', () => {
```

Comment:
I think we want just one level.

Reply:
CW events have a section called "details" which contains most pieces of information. We do not want to stringify it as a whole, I think. CW event: { region: ..., account: ... details: { instance_id:... , .... } } , etc. We may consider calling the helper function separately for it and merging the objects?